src/lang/en.json

```diff
@@ -135,6 +135,9 @@
         "ULC": "BC Unlimited Liability Company",
         "CUL": "BC Unlimited Liability Company",
         "C": "BC Limited Company",
+        "A": "Extraprovincial Company",
+        "XCR": "Extraprovincial Limited Company",
+        "XL": "Extraprovincial Limited Partnership",
```

Comment:
According to https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts, `A` is an expraprovincial company.

I added the other 2 because that's the legal type in a couple of NRs I created. (It's not a complete list but we can add the others later.)

![image](https://github.com/user-attachments/assets/f3853537-9155-4803-b177-a0f3b0357bf7)

Reply:
Before:
![image](https://github.com/user-attachments/assets/eceabb32-1dcb-4687-910b-045abf8c0c71)

After:
![image](https://github.com/user-attachments/assets/46aa72b9-1134-45f0-832d-0faacb4d5ea9)